integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml

```diff
@@ -9,22 +9,30 @@
 # See the NOTICE file distributed with this work for information regarding copyright ownership.
 #
 
+{{ $masterCount := int .Values.master.count }}
+{{ $defaultMasterName := "alluxio-master-0" }}
+{{- $isSingleMaster := eq $masterCount 1 }}
+{{- $isHaEmbedded := and (eq .Values.journal.type "EMBEDDED") (gt $masterCount 1) }}
+{{- $release := .Release }}
+{{- $name := include "alluxio.name" . }}
+{{- $fullName := include "alluxio.fullname" . }}
+{{- $chart := include "alluxio.chart" . }}
```

Comment:
I have to use local variables like this because in `{{range}}{{end}}` loops the variable tree is different.

Reply:
sgtm